src/DataWrangling/ECCO/ECCO_metadata.jl

```diff
@@ -33,25 +24,28 @@ end
 
 Base.show(io::IO, metadata::ECCOMetadata) = 
     print(io, "ECCOMetadata:", '\n',
-    "├── name: $(metadata.name)", '\n',
-    "├── dates: $(metadata.dates)", '\n',
-    "├── version: $(metadata.version)", '\n',
-    "└── dir: $(metadata.dir)")
+    "├── name: ", metadata.name, '\n',
+    "├── dates: ", datestr(metadata), '\n',
+    "├── version: ", versionstr(metadata), '\n',
+    "└── dir: ", metadata.dir)
+
+versionstr(md::ECCOMetadata) = string(md.version)
 
-Base.summary(md::ECCOMetadata{<:Any, <:ECCO2Daily})   = "ECCO2Daily $(md.name) metadata ($(first(md.dates))--$(last(md.dates)))"
-Base.summary(md::ECCOMetadata{<:Any, <:ECCO2Monthly}) = "ECCO2Monthly $(md.name) metadata ($(first(md.dates))--$(last(md.dates)))"
-Base.summary(md::ECCOMetadata{<:Any, <:ECCO4Monthly}) = "ECCO4Monthly $(md.name) metadata ($(first(md.dates))--$(last(md.dates)))"
+const AnyDateTime = Union{AbstractCFDateTime, Dates.AbstractDateTime}
+const ECCOMetadatum = ECCOMetadata{<:AnyDateTime}
 
-Base.summary(md::ECCOMetadata{<:AbstractCFDateTime, <:ECCO2Daily})   = "ECCO2Daily $(md.name) metadata at $(md.dates)"
-Base.summary(md::ECCOMetadata{<:AbstractCFDateTime, <:ECCO2Monthly}) = "ECCO2Monthly $(md.name) metadata at $(md.dates)"
-Base.summary(md::ECCOMetadata{<:AbstractCFDateTime, <:ECCO4Monthly}) = "ECCO4Monthly $(md.name) metadata at $(md.dates)"
+datestr(md::ECCOMetadata) = string(first(md.dates), "--", last(md.dates))
+datestr(md::ECCOMetadatum) = string(md.dates)
+
+Base.summary(md::ECCOMetadata) = string("ECCOMetadata{", versionstr(md), "} of ",
+                                        md.name, " for ", datestr(md))
 
 """
     ECCOMetadata(name::Symbol; 
-                 dates = DateTimeProlepticGregorian(1993, 1, 1),
+                 dates = DateTime(1993, 1, 1),
```

Comment:
Is the "TimeProlepticGregorian" not needed?

Reply:
`DateTime` is on the proleptic gregorian calendar. The other times from `CFTime` are necessary because they are not proleptic gregorian, however.